docs/DeveloperGuide.md

```diff
@@ -1,38 +1,212 @@
-# Developer Guide
+# WildWatch Developer Guide 🦏
+
```

Comment:
Overall good job. The DG has more to be completed so I don't have much comments to make. No issues with structure and flow. Nice that you have a table of contents and an option to take reader back to top. Cute emojis as well :). Don't forget to include class diagrams as well. I am sure you guys have plans to add in individual commands, explaining the design considerations and implementations.